### PR TITLE
chore: update deps & imporve readme

### PR DESCRIPTION
**Pull request by zhmushan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,47 @@ we currently supports the following SQL drives:
...
+## Installation
+
+```sh
+npm i casbin-basic-adapter
+```
+
+## Simple Example
+
+```ts
+import casbin from 'casbin';
```
Comment (nodece) [inline README.md]: please use `import { newEnforcer  } from 'casbin';` or `import * as casbin from 'casbin';`.

Comment (zhmushan) [reply README.md]: done.

**Comment by hsluoyz** [comment]
@zhmushan please add this adapter to our official docs.

**Comment by zhmushan** [comment]
Okey

**Review by nodece — APPROVED** [review]

**Comment by hsluoyz** [comment]
Add badges.

**Pull request merged**